sacrebleu.py

```diff
@@ -1468,7 +1468,7 @@ def main():
                                          #epilog = 'Available test sets: ' + ','.join(sorted(DATASETS.keys(), reverse=True)),
                                          formatter_class=argparse.RawDescriptionHelpFormatter)
     arg_parser.add_argument('--test-set', '-t', type=str, default=None,
-                            help='the test set to use (see also --list)')
+                            help='the test set to use (see also --list) or a comma-separated list of test sets to be concatenated')
     arg_parser.add_argument('-lc', action='store_true', default=False,
```

Comment:
See note below, this could be space-delimited if we use builtin argparse primitives.